### PR TITLE
NO-ISSUE - Fetch all masters from primary and secondary networks when IPv6 is set with None platform.

### PR DESCRIPTION
**Pull request by eliorerz**
/cc @tsorya @YuviGold

### CODE REVIEW

**Comment by eliorerz** [comment]
@YuviGold e2e-metal-assisted-none-ipv6 is not supported at the moment.
/override e2e-metal-assisted-none-ipv6

File: discovery-infra/test_infra/helper_classes/nodes.py [hunk 1]
```diff
@@ -167,7 +167,10 @@ def set_hostnames(self, cluster, nodes_count: int, is_ipv6: bool, is_static_ip:
             # set to valid hostname
             # TODO - NodeController has no `params` and `tf` attributes
             network_name = self.controller.params.libvirt_network_name
+            secondary_network_name = self.controller.params.libvirt_secondary_network_name
             libvirt_nodes = utils.get_libvirt_nodes_from_tf_state(network_name, self.controller.tf.get_state())
+            libvirt_nodes.update(utils.get_libvirt_nodes_from_tf_state(secondary_network_name,
```
Comment (ori-amizur) [inline discovery-infra/test_infra/helper_classes/nodes.py:172]: Seems there is a mismatch. _get_libvirt_nodes_from_tf_state_ receives network_names collection as first parameter, and here we pass a single network.

**Review by ori-amizur — CHANGES_REQUESTED** [review]

File: discovery-infra/test_infra/helper_classes/nodes.py (hunk 1 above)
Comment (eliorerz) [reply discovery-infra/test_infra/helper_classes/nodes.py:172]: @ori-amizur You right, didn't noticed it. The original implementation expect it to be an iterable object but here we pass string (luckily strings in python are iterables so it worked so far).

**Pull request merged**